data/persons.txt

```diff
@@ -1030,11 +1049,17 @@ outfit "Shooting Star Hit"
 		"reload" 1
 		"piercing" 1
 		"hull damage" 293
+	description "There appears to have been various attempts in the past to discover the workings of this outfit, but the debate is still ongoing wether this can be considered a device or not."
```

Comment:
```suggestion
	description "There appear to have been several prior attempts to discern the mechanics of this outfit, but the debate is still ongoing whether this can be considered a device or not."
```